sql/moz-fx-data-shared-prod/accounts_backend/nonprod_accounts/view.sql

```diff
@@ -0,0 +1,17 @@
+CREATE OR REPLACE VIEW
+  `moz-fx-data-shared-prod.accounts_backend.nonprod_accounts`
```

Comment:
Since this is an abridged/redacted version of the accounts data, should we give the view a more distinctive name that indicates that?

Or what is the plan for exposing the sensitive data columns as necessary with appropriate restrictions?

Reply:
Restricted columns will be accessed directly via the table in `_external` dataset that will be accessible by members of `accounts-confidential` group. These columns won't be exposed in Looker.

Having said that the name sounds fine to me but I'm open to suggestions.